test/integration/kubernetes_deploy_test.rb

```diff
@@ -1048,16 +1107,15 @@ def test_streams_unmanaged_pod_logs_when_only_one
   def test_bad_container_on_daemon_sets_fails
     assert_deploy_failure(deploy_fixtures("invalid", subset: ["crash_loop_daemon_set.yml"]))
     num_ds = expected_daemonset_pod_count
-    assert_logs_match_all([
+    expect_log = [
       "Failed to deploy 1 resource",
       "DaemonSet/crash-loop: FAILED",
       "crash-loop-back-off: Crashing repeatedly (exit 1). See logs for more information.",
       "Final status: #{num_ds} updatedNumberScheduled, #{num_ds} desiredNumberScheduled, 0 numberReady",
-      "Events (common success events excluded):",
```

Comment:
If events aren't being printed something is wrong, I don't see why your code change would impact this

Reply:
may flacky, let me verify

Reply:
actually not flacky test, they are event log, same reason as https://github.com/Shopify/kubernetes-deploy/pull/553/files#r325686855